### PR TITLE
Update GMT info and listing

### PR DESCRIPTION
**Pull request by osuChaos**
Add information regarding how to become a GMT candidate, update team members and languages moderated sections

### CODE REVIEW

File: wiki/People/The_Team/Global_Moderation_Team/en.md [hunk 1]
```diff
@@ -2,13 +2,31 @@
...
 Becoming a member of the **Global Moderation Team** requires a notable amount of contribution. Potential members are often noticed for their willingness to help other users, activity within the community, and good behaviour. The ability to speak languages other than English is strongly preferred. Candidates are discussed within the team to determine their potential addition as a team member.
 
-## Roles and responsibilities
+### How to be considered for the GMT
+
+The most important factors for consideration are behaviour, activity, understanding of the rules, and patience:
+
+1. Demonstrate good behaviour by being polite and treating other users with respect. Keeping your account free of any serious silences or issues is essential.
+2. Demonstrate in-game activity by participating in the chat and/or the forums. Report any misbehaviour and problems via the report system. Answer questions and assist others with issues.
+3. Have a proper understanding of [the rules](https://osu.ppy.sh/help/wiki/Rules) and how they are applied. It is important to know what is acceptable within osu! and what is not.
```
Comment (MegaApplePi) [inline wiki/People/The_Team/Global_Moderation_Team/en.md]:
```suggestion
3. Have a proper understanding of [the rules](/wiki/Rules) and how they are applied. It is important to know what is acceptable within osu! and what is not.
```

File: wiki/People/The_Team/Global_Moderation_Team/en.md [hunk 2]
```diff
@@ -2,13 +2,31 @@
...
+
+### Some things not to do
 
-The Global Moderation Team is responsible for the welfare of the chat/forum and takes care of interactions with the community. They should be approached if you have any questions pertaining the moderation aspect of the game, which would include (but is not limited to) issues with chat (silences, harassment) and the forum (spam).
+1. Do not continually ask the GMT about your standing as a potential candidate. The GMT does not look to add anyone who is contributing as a means to an end. An altruistic mindset is key.
+2. Do not report every little thing in chat. It is an important skill to know what is worth reporting and what is not.
+3. Do not answer other people's questions without knowing whether said answer is correct or not, particularly on the forums. Suggestions are welcome, but misinforming people isn't helpful to anyone. If you don't know the solution to a problem, let someone else answer and learn from them.
```
Comment (MegaApplePi) [inline wiki/People/The_Team/Global_Moderation_Team/en.md]:
```suggestion
3. Do not answer other people's questions without knowing whether said answer is correct or not, particularly on the forums. Suggestions are welcome, but misinforming people is not helpful to anyone. If you do not know the solution to a problem, let someone else answer and learn from them.
```

**Comment by osuChaos** [comment]
Might be reasonable to remove the role HappyStick and mm201, but I don't think I have any say on asking ppy and smoogi to have the GMT role removed since I assume they have it for development related purposes. However they don't do any community moderation so it doesn't really make any sense imo to have them listed on the wiki page

**Comment by cl8n** [comment]
doesn't hurt to try, I think that'd be best because if they aren't doing moderation / GMT work then it's equally misleading to show them on the Groups page. the only reason we list users here is cuz some info is missing from the Groups page; the users should always be 1:1 and we occasionally verify this by program too

**Comment by cl8n** [comment]
no reply here for a while so i fixed & rewording some things myself. most of it was already good 👍

**Pull request merged**